src/stories/Library/dropdown/dropdown.scss

```diff
@@ -8,6 +8,14 @@
   }
 }
 
+.dropdown__half-on-desktop {
```

Comment:
Is it relevant to describe what is happening when hitting the desktop breakpoint? Wouldn't: `.dropdown_desktop` do it? If, for some reason, it would be decided that it should have a third witdth in the future you would have to rename the class :)

Reply:
I'll rename